File: aurafit-frontend/src/Models/DTOS/PostReviewRequestDTO.ts
import {Rating} from "../Enums/Rating.ts";
import {Product} from "../Product.ts";


export class PostReviewRequestDTO {
    content: string;
    rating: Rating;
    product: Product;

    constructor(content: string, rating: Rating, product: Product) {
        this.content = content;
        this.rating = rating;
        this.product = product;
    }
}

File: AuraFit-BackEnd/src/main/java/app/aurafitbackend/DTOS/PostReviewRequestDTO.java
package app.aurafitbackend.DTOS;

import app.aurafitbackend.Beans.Product;
import app.aurafitbackend.Enums.Rating;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
public class PostReviewRequestDTO {

    private String content;
    private Rating rating;
    private Product product;

}
